just-install.json

```diff
@@ -464,6 +464,16 @@
       },
       "version": "0.2"
     },
+    "fences": {
+      "installer": {
+        "kind": "innosetup",
```

Comment:
since fences is using [SetupFactory](https://setupfactory.com/), its using parameters [similar to NSIS](https://www.indigorose.com/webhelp/suf9/Program_Reference/Command_Line_Options.htm#Install_Command_Line_Options)

Reply:
since fences requires a licence to finish the installation, it should look like this:

````
    "fences": {
      "installer": {
        "kind": "nsis",
        "interactive": true,
        "options": {
          "extension": ".exe"
        },
        "x86": "https://store.stardock.com/download/3875"
      },
      "version": "latest"
    },
````